dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -18,7 +18,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 # The file referenced (`/usr/share/dpkg/no-pie-compile.specs`) is in the `libdpkg-perl` package on Debian
 RUN apt-get update   && \
     apt-get install -y  \
-        curl            \
+        libpci3 libsensors5 \
+        python3.7 python3-pip \
```

Comment:
>python3.7 [](start = 8, length = 9)

Why not install python3 and python3-dev?

Reply:
Done